app/src/main/java/com/jerboa/ui/components/home/HomeActivity.kt

```diff
@@ -313,20 +322,22 @@ fun MainPostListingsContent(
             onPostClick = { postView ->
                 appState.toPost(id = postView.post.id)
             },
-            onSaveClick = { postView ->
-                account.doIfReadyElseDisplayInfo(
-                    appState,
-                    ctx,
-                    snackbarHostState,
-                    scope,
-                    siteViewModel,
-                ) {
-                    homeViewModel.savePost(
-                        SavePost(
-                            post_id = postView.post.id,
-                            save = !postView.saved,
-                        ),
-                    )
+            onSaveClick = remember {
```

Comment:
These remembers can indeed be removed I forgot to do so when I enabled Strong Skipping. But before these were required to avoid being constantly recomposed.

Reply:
Also these function don't need to be keyed with postview. They are pure, and receive postview as argument thus do not need to be recreated when the postview changes. It will receive the new postview as argument.